### ISSUE TITLE
If the cargo index is removed install-update fails to run

### ISSUE THREAD

**Issue by xbb** [nabijaczleweli/cargo-update#150]
Clearing the cargo index (manually or for example via cargo-cache crate) breaks cargo install-update -l/-a etc...

```
Couldn't get package repository: Couldn't read /home/user/.cargo/registry/index/github.com-1ecc6299db9ec823 (index directory for https://github.com/rust-lang/crates.io-index): No such file or directory (os error 2).
```

A work around is to rebuild the index with `cargo search --limit 0` or `cargo search <crate>`

**Comment by nabijaczleweli** [comment]
Can you try current `master` (at least 5420022d7ade463bf77fab75b2c62fed9364b3bd)?

**Comment by xbb** [comment]
Just tested and it works fine!

Thank you

**Comment by nabijaczleweli** [comment]
Cool, released in [`v5.0.0`](https://github.com/nabijaczleweli/cargo-update/releases/tag/v5.0.0)!
